var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -888,7 +888,9 @@ def fix_package_config(self):
             content += " " + self.spec["lapack"].libs.ld_flags
             content += " " + self.spec["fftw-api"].libs.ld_flags
 
-            if (self.spec["fftw-api"].name == "fftw") and ("+openmp" in self.spec["fftw"]):
+            if (self.spec["fftw-api"].name in ["fftw", "amdfftw"]) and self.spec[
+                "fftw-api"
+            ].satisfies("+openmp"):
```

Comment:
```suggestion
            fftw = self.spec["fftw-api"]
            if fftw.name in ["fftw", "amdfftw"] and fftw.satisfies("+openmp"):
```
Not a big deal, but this might be easier to read.

Reply:
That makes sense to me